src/figuresClasses.ts

```diff
@@ -1,19 +1,102 @@
+
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle'| 'circle' |'rectangle';
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea():number
 
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  color: Color;
+
+  a:number;
+
+  b:number;
+
+  c:number;
+
+  shape:Shape = 'triangle';
+
+  constructor(color:Color, a:number, b:number, c:number) {
```

Comment:
you didn't fix the comment
rewrite receiving and setting props in constructor using private modifier
example: `public color: Color, `